go.mod

```diff
@@ -4,8 +4,7 @@ go 1.13
 
 require (
 	github.com/alexedwards/scs v1.4.1
-	github.com/bradleyfalzon/ghinstallation v1.1.1
-	github.com/google/go-github/v29 v29.0.3 // indirect
+	github.com/bradleyfalzon/ghinstallation v1.1.2-0.20210508092528-3386a2062511
```

Comment:
It'd be nice to get a real release of the lib so we don't need to depend on the pseudo version (https://github.com/bradleyfalzon/ghinstallation/issues/50)